addons/mail/static/src/core/common/thread_service.js

```diff
@@ -98,13 +98,15 @@ export class ThreadService {
             this.rpc("/discuss/channel/set_last_seen_message", {
                 channel_id: thread.id,
                 last_message_id: newestPersistentMessage.id,
-            }).then(() => {
-                this.updateSeen(thread, newestPersistentMessage.id);
-            }).catch((e) => {
-                if (e.code !== 404) {
-                    throw e;
-                }
-            });
+            })
+                .then(() => {
```

Comment:
Does the indentation seem weird?

Reply:
prettier is stubborn and wants that formatting